.buildkite/pipeline.yml

```diff
@@ -5,7 +5,36 @@ steps:
       - "make compile"
       - "make ci"
     name: ":hammer: build"
+    key: "tests"
```

Comment:
What is this key for?

Reply:
Well, eventually I'd like to make building the image on having a clean test run, so this just sets the stage for that. Down the road we can add a

```yaml
  depends_on: tests
```

line in there.